rts/lra/test/pom.xml

```diff
@@ -196,16 +224,113 @@
               </execution>
             </executions>
           </plugin>
+
+          <!-- Makes sure that the two Wildfly instances use different standalone XML configurations -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>${version.antrun.plugin}</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <phase>generate-test-resources</phase>
+                <configuration>
+                  <target>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.coordinator.xml.filename}"></copy>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.participant.xml.filename}"></copy>
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <!--
+            This profile needs to kick in when Wildfly is used to run this TCK.
+            The reason is that lra-coordinator is not in the LRA specification
+            and, therefore, a scenario with two deployments in a single instance
+            of Wildfly (or distributed in two Wildfly instances) is not compatible
+            with the LRA TCK.
+          -->
+          <plugin>
+            <groupId>org.wildfly.plugins</groupId>
+            <artifactId>wildfly-maven-plugin</artifactId>
+            <version>${version.wildfly-maven-plugin}</version>
+            <configuration>
+              <skip>${skip.wildfly.plugin}</skip>
+              <!-- To start Wildfly in debug mode
```

Comment:
I was wrong in explanation of the point, I assume.
I was thinking only about the `To start WildFly in debug mode`. I think it should not be required to start coordinator in debug by uncommenting some lines in `pom.xml`. It should be possible to run just `mvn test -Parq -Ddebug.coordinator` and being able to connect with debugger.

If the debug change makes some troubles it can be postponed to some later change, like when I will need to debug it I will for sure add something like that (I would be angry if I need to comment and uncomment when investigating.)

Reply:
Sorry, I completely misunderstood your comment. I used the debug option of `wildfly-maven-plugin` for fixing some issues I was facing (i.e. the problem with the Container Filter and the auto-discovery feature of JAX-RS). I also think that the option to debug `wildfly-maven-plugin` should be removed as `lra-coordinator` should not be debugged when used in integration tests. Thanks for highlighting this @ochaloup

Reply:
I consider it's handy to have a chance to debug the `lra-coordinator` when running the integration tests but it's fine as it is now.

Reply:
I removed the debug option as (IMO) `rts/lra/test` should only test `lra-coordinator` and not debug it. From my POV, `lra-coordinator` should be debugged in `lra-coordinator-jar`. Of course, this is only my opinion so feel free to ask me to allow debugging here :-)